locales/de.json

```diff
@@ -2290,7 +2290,10 @@
 			}
 		},
 		"footer": {
-			"ariaLabel": "footer",
+			"ariaLabel": {
```

Comment:
please use aria_label, according to 
https://docs.schul-cloud.org/pages/viewpage.action?pageId=113643606
![image](https://user-images.githubusercontent.com/40116220/89407664-767bbc00-d71f-11ea-952d-aa9d0e6da9cd.png)